CMakeLists.txt

```diff
@@ -98,4 +98,12 @@ if(UTF8PROC_ENABLE_TESTING)
   target_link_libraries(normtest utf8proc)
   add_test(utf8proc.testgraphemetest graphemetest data/GraphemeBreakTest.txt)
   add_test(utf8proc.testnormtest normtest data/NormalizationTest.txt)
+  
+  if(DEFINED ENV{LIB_FUZZING_ENGINE})
```

Comment:
I'd rather require the user to explicitly pass `-DLIB_FUZZING_ENGINE=...` than use an environment variable, since environment variables are too vulnerable to being set accidentally.   The `ossfuzz.sh` script can then pass `-DLIB_FUZZING_ENGINE="$LIB_FUZZING_ENGINE"` using the environment variable.

Reply:
Updated to use a build option.